routes/v1/account.js

```diff
@@ -105,6 +105,24 @@ router.get('/info', auth.jwt, safeHandler(async (req, res) => {
     return res.status(constants.STATUS_CODES.OK).json(info);
 }));
 
+router.get('/settings', auth.jwt, safeHandler(async (req, res) => {
+    const settings = await authLogic.getSettings();
+
+    return res.status(constants.STATUS_CODES.OK).json(settings);
+}));
+
+router.post('/settings/update', auth.jwt, safeHandler(async (req, res) => {
```

Comment:
To be RESTful the resource should be constant and the HTTP verb should describe the action.

So instead of 

```
GET /settings
POST /settings/update
```

it should just be

```
GET /settings
POST /settings
```

Technically it should actually be PUT not POST since the operation is idempotent and is updating a resource not creating a new one.

And technically technically it should be PATCH not PUT since you're only replacing part of the resource not overriding the entire resource.

But we haven't followed REST that strictly anywhere else so no worries about PUT/PATCH, sticking with POST is fine for consistency. But we should definitely use constant resource URLs.

Reply:
That makes sense, I will change the route.